core/kernel/lockdep.c

```diff
@@ -265,12 +269,19 @@ static void lockdep_print_edge_info(uintptr_t from __maybe_unused,
 				    struct lockdep_edge *edge)
 {
 	uintptr_t __maybe_unused to = edge->to->lock_id;
+	const char *at_msg = "";
```

Comment:
`__maybe_unused`?

Reply:
In fact they're always used in the `if` so it doesn't seem necessary (although TBH I have not tried to build with lockep and no logs).

Reply:
Yeah, I was thinking of `-Wunused-but-set-variable`.

Reply:
OK I will add `__maybe_unused`